internal/api/web/views/components/dashboard_components/dashboard_home_page/quiz_tile.templ

```diff
@@ -0,0 +1,25 @@
+package dashboard_components
+
+import "github.com/Molnes/Nyhetsjeger/internal/data/quizzes"
+
+templ QuizTile(quiz quizzes.Quiz) {
+	<a
+		class="mr-4 mb-4 bg-gray-200 shadow-md rounded-lg max-w-56 overflow-hidden flex flex-col hover:shadow-lg transition duration-100 ease-in-out transform hover:-translate-y-1 hover:scale-105"
```

Comment:
Instead of using margin right and bottom, consider having the parent be a flexbox and then use `gap`  instead.

Reply:
3a8e6308a496c5e3375d344f65a6c2f2cd81220e